addons/rose/addon/styles/rose/components/table/_table.scss

```diff
@@ -35,7 +39,33 @@
     }
   }
 
+  // Grow to full available width
+  .rose-table-cell-expand {
+    width: 100%;
+
+    & > * {
```

Comment:
For performance reasons, we should avoid `*` selectors.  What is the purpose of this block and can it be rewritten?